### DISCUSSION TITLE
Should I not use `cva` in a SPA application?

### DISCUSSION THREAD

**Discussion by binajmen** [joe-bell/cva#280] [Q&A] [answered]
The documentation says:
> Although cva is a tiny library, it's best to use in a SSR/SSG environment – your user probably doesn't need this JavaScript, especially for static components.

src: https://cva.style/docs/getting-started/variants#creating-variants

I want to use it for a SPA application. Is it wrong?

**Comment by joe-bell** [comment, marked as answer]
You can use it!

**Reply by binajmen** [reply]
I guess I can trust you ;) Out of curiosity, is this info panel only exists to warn about the small bundle size impact, or is there a perf impact as well?
